### PR TITLE
Google Assistant SDK: Document conversation agent

### PR DESCRIPTION
**Pull request by tronikos**
## Proposed change

Document Google Assistant SDK conversation agent support

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/85499
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/google_assistant_sdk.markdown [hunk 1]
```diff
@@ -133,3 +134,22 @@ data:
     - bedroom
     - basement
 ```
+
+## Conversation agent
+
+In configuration.yaml add:
+```yaml
+conversation:
+```
```
Comment (frenck) [inline source/_integrations/google_assistant_sdk.markdown]:
````suggestion
In `configuration.yaml` add:
```yaml
conversation:
```
````

File: source/_integrations/google_assistant_sdk.markdown [hunk 2]
```diff
@@ -133,3 +134,22 @@ data:
...
+
+In the configure options of the integration enable the conversation agent and then you can converse with Google Assistant by pressing the microphone in the frontend (supported browsers only):
```
Comment (frenck) [inline source/_integrations/google_assistant_sdk.markdown]:
```suggestion
In the configure options of the integration, enable the conversation agent and then you can converse with Google Assistant by pressing the microphone in the frontend (supported browsers only):
```

File: source/_integrations/google_assistant_sdk.markdown [hunk 3]
```diff
@@ -133,3 +134,22 @@ data:
...
+
+![google_assistant_sdk_conversation](https://user-images.githubusercontent.com/9987465/211274341-85dfe8a2-840f-4670-9e82-01777318e463.png)
```
Comment (frenck) [inline source/_integrations/google_assistant_sdk.markdown]: Image must be included in this repository and embedded locally (and thus not hosted in GitHub user images).

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/google_assistant_sdk.markdown (hunk 3 above)
Comment (tronikos) [reply source/_integrations/google_assistant_sdk.markdown]: Done

File: source/_integrations/google_assistant_sdk.markdown [hunk 4]
```diff
@@ -133,3 +134,22 @@ data:
...
+In the configure options of the integration, enable the conversation agent and then you can converse with Google Assistant by pressing the microphone in the frontend (supported browsers only):
+
+![Screenshot Conversation](/images/integrations/google_assistant_sdk/conversation.png)
+
+Or by calling the `conversation.process` service with the transcribed text:
+
+```yaml
+service: conversation.process
+data:
+  text: dim the family room lights
```
Comment (frenck) [inline source/_integrations/google_assistant_sdk.markdown]:
```suggestion
  text: "Dim the family room lights"
```

File: source/_integrations/google_assistant_sdk.markdown [hunk 5]
```diff
@@ -27,6 +27,7 @@ This integration allows:
   - Someone is at the front door
   - Smoke detected in the master bedroom
   - Water leak detected in the master bathroom
+- Having a conversation with Google Assistant using the [conversation](https://www.home-assistant.io/integrations/conversation/) integration via text or voice.
```
Comment (frenck) [inline source/_integrations/google_assistant_sdk.markdown]:
```suggestion
- Having a conversation with Google Assistant using the [conversation](/integrations/conversation/) integration via text or voice.
```

**Review by frenck — APPROVED** [review]
Thanks, @tronikos 👍 

../Frenck

**Pull request merged**